### DISCUSSION TITLE
Could an image be inserted in the main tab?

### DISCUSSION THREAD

**Discussion by Natii9898** [globaleaks/globaleaks-whistleblowing-software#3872] [General]
Good afternoon, I have the following question.
I wanted to put an image in the container in the main part.
<img width="427" alt="image" src="https://github.com/globaleaks/GlobaLeaks/assets/153637801/fc50bea1-f87c-404f-9fb4-db3aa886caf0">

I want to put it under that text which is "We recommend that you access the "Preferences" section to download your "Account Recovery Key" and store it securely. This key will be necessary to recover your access to the platform and your data. in case you forget your password."

I have a question if that could be possible, I tried it with javascript by putting this var img = document.createElement("img");
img.src = "transports.png";
img.alt = "Image description";
document.getElementById("Content").appendChild(img);
in img.src it tests as many paths of the server where I have it installed as images per internet url

I run into the problem that the url does not allow images to be uploaded since for security reasons etc. it does not upload them and neither does the server and the server routes.

I thought about uploading the image to globaleaks in the files section but I have another question, so could I display it via javascript? Or where for example do you save those files on the server and in that case they can be extracted and displayed using javascript?

<img width="476" alt="image" src="https://github.com/globaleaks/GlobaLeaks/assets/153637801/81c8d9c1-1ab3-4313-9943-5f007bad4b39">

Thank you

**Comment by evilaliv3** [comment]
Uploaded files are put available on the path /s/filename

Try to upload a file and you will be given a button enabling to download the file or link it within your CSS

**Reply by Natii9898** [reply]
Yes but through css how can I do it? And in any case the entire path where said files will be saved in case I can load the image that I saved there.
